docs/api.rst

```diff
@@ -47,6 +47,12 @@ SELIC
 .. autoclass:: calculadora_do_cidadao.Selic
    :members:
 
+Família CPI
+~~~~~
```

Comment:
In reStructuredText the good practice is to have the same number of _underline_ character as the title. Can we fix that?

Reply:
Also: can we keep this page and its adapters/families organized al[habetically for better maintainability?